src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ *  (see an example in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newArray = [];
+
+  const properties = sourceString.split(';')
+    .map(item => item.trim().split(':'))
+    .filter(item => !item.includes(''));
+
+  properties.forEach(value =>
+    newArray.push([value[0].trim(),
+      value[1].trim()]));
```

Comment:
legit approach, but let's use de-structuring:
```suggestion
  properties.forEach([property, value] =>
    newArray.push([property.trim(),
      value.trim()]));
```